assets/javascript/app.js

```diff
@@ -5,35 +5,44 @@ var incorrectanswer;
 var unanswered;
 var userpicked;
 //  When the resume button gets clicked, execute the run function.
-$("#startbutton").on("click", run);
-
-function myFunction(optradio1) {
-
-    $("result1").value = optradio1;
-};
-
-
-//  Clearing the intervalId prior to setting our new intervalId will not allow multiple instances.
-function run() {
-    clearInterval(intervalId);
-    intervalId = setInterval(decrement, 1000);
-    console.log('intervalId', intervalId);
-}
-
-
-
-
-//  The decrement function.
-function decrement() {
-    //  Decrease number by one.
-    number--;
-    //  Show the number in the #show-number tag.
-    $("#timeout").text(number);
-    //  Once number hits zero...
-    if (number === 0) {
-        $("#questionnaire").hide();
+$(document).ready(function() {
+
+    $("#startbutton").on("click", run);
+    
+    $(".radioBtn").on('click', function(event) {
+       // console.log('event clicked', event);
+       if (event.target.name === 'optradio1') {
+        $("#result1").text(event.target.value);
```

Comment:
Here we are checking to see if it's the first question.  If so, I'm putting the response into the element with the id='result1'.  You may want to put it somewhere else.  Also may want to add logic here to see if they clicked on the correct answer.   Or maybe you want to do that when a "Submit" button is clicked on the very bottom.   If so, you'll have to add that.